golden/gas-model/golden

```diff
@@ -1 +1 @@
tains medNumber medNumberList)",140],["(contains smallNumber smallNumberList)",140],["(contains \"a1\" longOjectMap)",140],["(contains \"a1\" medOjectMap)",140],["(contains \"a1\" smallOjectMap)",140],["(contains \"alongNumber\" longString)",140],["(contains \"amedNumber\" medString)",140],["(contains \"asmallNumber\" smallString)",140],["(format-time \"%F\" (time \"2016-07-22T12:00:00Z\"))",144],["(format-time \"%Y-%m-%dT%H:%M:%S%N\" (time \"2016-07-23T13:30:45Z\"))",144],["(>= longNumber longNumber)",140],["(>= medNumber medNumber)",140],["(>= smallNumber smallNumber)",140],["(>= longNumber.0 longNumber.0)",140],["(>= medNumber.0 medNumber.0)",140],["(>= smallNumber.0 smallNumber.0)",140],["(>= longString longString)",140],["(>= medString medString)",140],["(>= smallString smallString)",140],["(>= (time \"2016-07-22T12:00:00Z\") (time \"2018-07-22T12:00:00Z\"))",144],["(take 1 longNumberList)",141],["(take 1 medNumberList)",141],["(take 1 smallNumberList)",141],["(take -1 longNumberList)",141],["(take -1 medNumberList)",141],["(take -1 smallNumberList)",141],["(take longEscapedStrList longOjectMap)",141],["(take medEscapedStrList medOjectMap)",141],["(take smallEscapedStrList smallOjectMap)",141],["(take [\"a1\"] longOjectMap)",141],["(take [\"a1\"] medOjectMap)",141],["(take [\"a1\"] smallOjectMap)",141],["(constantly 0 \"firstIgnore\")",139],["(constantly 0 \"firstIgnore\" \"secondIgnore\")",139],["(constantly 0 \"firstIgnore\" \"secondIgnore\" \"thirdIgnore\")",139],["(with-default-read\n   accounts.accounts\n   \"someId\"\n   { \"balance\": 1.0 }\n   { \"balance\":= bal }\n   bal\n)",153],["(create-module-guard \"test\")",139],["(pact-id)",139],["(enforce true \"some-error-message\")",139],["(describe-table accounts.accounts)",238],["(identity longNumberList)",140],["(identity medNumberList)",140],["(identity smallNumberList)",140],["(enforce-keyset 'some-loaded-keyset)",147],["(& longNumber longNumber)",139],["(& medNumber medNumber)",139],["(& smallNumber smallNumber)",139],["(update accounts.accounts\n        \"someId\"\n        { \"balance\": 10.0 }\n)",263],["(resume longBinding a1)",10002],["(resume medBinding a1)",102],["(resume smallBinding a1)",12],["(resume longBinding a1) with provenance",10140],["(resume medBinding a1) with provenance",240],["(resume smallBinding a1) with provenance",150],["(keyset-ref-guard \"some-loaded-keyset\")",145],["(add-time (time \"2016-07-22T12:00:00Z\") 15)",143],["(read-integer \"amount\") with amount=longNumber",139],["(read-integer \"amount\") with amount=medNumber",139],["(read-integer \"amount\") with amount=smallNumber",139],["(typeof longOjectMap)",140],["(typeof medOjectMap)",140],["(typeof smallOjectMap)",140],["(typeof longString)",140],["(typeof medString)",140],["(typeof smallString)",140],["(typeof longNumberList)",140],["(typeof medNumberList)",140],["(typeof smallNumberList)",140],["(typeof longNumber)",140],["(typeof medNumber)",140],["(typeof smallNumber)",140],["(pact-version)",139]]
```

Comment:
The only changes are removed '\n' characters and changing:

```
("(interface my-interface\n  (defun say-hello:string (name:string))\n)\n\n(module some-random-module GOV\n  (implements my-interface)\n  (defcap GOV ()\n    true\n  )\n\n  (defun say-hello:string (name:string)\n    name)\n)\n",219)
```

to

```
("(interface my-interface\n  (defun say-hello:string (name:string))\n)\n\n(module some-random-module GOV\n  (implements my-interface)\n  (defcap GOV ()\n    true\n  )\n\n  (defun say-hello:string (name:string)\n    name)\n)",218)
```

I confirmed that the latter change is caused by the change in the test input formatting and not by another side-effect of changes in this PR. I implemented the change in the test without neat-interpolation and was able to reproduce both of above results with the code changes of this PR.